src/main/java/de/anteiku/kittybot/events/OnGuildMessageEvent.java

```diff
@@ -18,6 +23,9 @@
 
 	@Override
 	public void onGuildMessageReceived(GuildMessageReceivedEvent event){
+		var content = event.getMessage().getContentRaw();
+		if (!content.isEmpty())
```

Comment:
{} :)

Reply:
you can just format the code with your code style then

Reply:
yus